.github/workflows/windows.yml

```diff
@@ -26,38 +25,10 @@ jobs:
       - name: Update gems
         shell: pwsh
         run: |
-          $docTypes = ConvertFrom-StringData (Get-Content ./doctypes.env -Raw)
-          gem install metanorma-cli metanorma-$($docTypes.TYPE)
-      - name: Use Node
-        uses: actions/setup-node@v1
-        with:
-          node-version: '8'
-      - name: Install Puppeteer
-        shell: pwsh
-        run: |
-          npm install -g puppeteer
-      - name: Use Python
-        uses: actions/setup-python@v1
-        with:
-          python-version: '2.7'
-          architecture: 'x64'
-      - name: Install xml2rfc
-        run: |
-          pip install xml2rfc
-      - name: Metanorma compile
-        shell: pwsh
+          gem install bundler
+          bundle config --local path vendor/bundle
+          bundle update
```

Comment:
@CAMOBAP795 There's no `Gemfile.lock` so there's no need to `bundle update`?

Reply:
Good catch! should we remove it from all CI configuration, AFAIR we decided to not commit `Gemfile.lock` anymore, right?

Reply:
Yes please! Let's get of them :wink:

Reply:
I will https://github.com/metanorma/metanorma-build-scripts/issues/36